mriqc/bin/mriqc_run.py

```diff
@@ -252,14 +269,30 @@ def main():
     # Process data types
     modalities = opts.modalities
 
-    dataset = collect_bids_data(
-        settings['bids_dir'],
-        modalities=modalities,
-        participant_label=opts.participant_label,
-        session=opts.session_id,
-        run=opts.run_id,
-        task=opts.task_id,
-    )
+    if opts.T1w_file:
+        dataset=dict()
```

Comment:
Well it break a few lines after (327) in the for loop
        for mod in modalities:
            if not dataset[mod]:

But if I properlie set modalities (to only ['T1w'] ), I do not need to define bold and T2w
I'll do the changes